MovieDemoApp/Views/Movie Detail/Movie Image and Ratings/ImageAndRatingsView.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  ImageAndRatingsView.swift
+//  MovieDemoApp
+//  Copyright © 2020 Deloitte Digital. All rights reserved.
+//
+
+import SwiftUI
+
+/// Movie Image and Ratings view combined
+struct ImageAndRatingsView: View {
+    var image: String
+
+    var body: some View {
+        ZStack(alignment: .bottomTrailing) {
+            MoviewImageView(image: image)
+            MovieRatingsView()
+        }
+
+    }
+}
+
+// View Modifier for custom side corner radius
+struct CornerRadiusStyle: ViewModifier {
```

Comment:
could we not use rounded rectangle object directly for this purposes? Any specific reason we used a view modifier?

Reply:
Rounded rectangle does not allow to add corner radius for specific size.
Using this modifier I was able to add corner radius to any specific side.